packages/core/database/package.json

```diff
@@ -37,6 +37,7 @@
     "knex": "2.4.0",
     "lodash": "4.17.21",
     "semver": "7.3.8",
+    "sqlite3": "5.1.2",
```

Comment:
It seems to have been added because it's used here. I've automated and fixed all this, so it might be weird in some places.
https://github.com/strapi/strapi/blob/78630d6da3e397cc4f3ecf075bc777dcb643addb/packages/core/database/lib/connection.js#L20-L24

Log: https://gist.github.com/smorimoto/6cac28bcdce5976f0ba42e8a5be2c417#file-safe-packages-json-L19

Reply:
Reading the code we can revert it to not being included. We test to see if it's installed before using that class and the class is also not exported 😊

Thanks for the explanation!

Reply:
Anytime, Indeed.